media-video/noad/noad-0.8.8.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 2021-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="Mark commercial breaks in VDR recordings"
+HOMEPAGE="https://github.com/madmartin/noad"
+SRC_URI="https://github.com/madmartin/noad/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+ffmpeg imagemagick libmpeg2"
+REQUIRED_USE="|| ( ffmpeg libmpeg2 )"
+
+DEPEND="
+	libmpeg2? ( media-libs/libmpeg2:= )
+	ffmpeg? ( media-video/ffmpeg )
+	imagemagick? ( media-gfx/imagemagick )"
+RDEPEND="${DEPEND}"
+BDEPEND="virtual/pkgconfig
+	${DEPEND}"
```

Comment:
Why is `DEPEND` in `BDEPEND`?

Reply:
> Why is `DEPEND` in `BDEPEND`?

Because I understood the meaning of BEDEPEND in a wrong way. It's fixed now.